orchestrator/index.js

```diff
@@ -0,0 +1,63 @@
+const request = require('request-promise')
+const logger = require('winston')
+const habanero = require('node-red-habanero')
+const habaneroSettings = require('../node_modules/node-red-habanero/settings')
+
+const config = require('../config/server')
+
+const HABANERO_ADMIN_ROOT = '/orchestrator'
+const HABANERO_NODE_ROOT = '/orchestrator-api'
+
+function getHabaneroBaseUrl(req){
+  var baseUrl = `${req.protocol}://${req.get('host')}${HABANERO_ADMIN_ROOT}/`
+  return baseUrl
+}
+
+function doHabaneroLogin(habaneroBaseUrl) {
+  return request({
+    uri: `${habaneroBaseUrl}auth/token`,
+    method: 'POST',
+    form: {
+      FROM_CONCARIA: 'true',
+      client_id: 'node-red-editor',
+      grant_type: 'password',
+      scope:'',
+      username: config.account.emailAddress,
+      password: config.account.password,
+      accountId: config.account.accountId,
+      XIVELY_ACCOUNT_USER_IDM_ID: config.account.idmUserId,
+      XIVELY_ACCOUNT_USER_BP_ID: config.account.blueprintUserId,
+      SALESFORCE_USER: config.salesforce.user,
+      SALESFORCE_PASSWORD: config.salesforce.pass,
+      SALESFORCE_TOKEN: config.salesforce.token
+    }
+  });  
+}
+
+function gotoHabanero(req, res){
+  var habaneroBaseUrl = getHabaneroBaseUrl(req);
+  doHabaneroLogin(habaneroBaseUrl).then(function(loginResult){
+    var ticket = new Buffer(loginResult).toString('base64')
+    var loginUrl = `${habaneroBaseUrl}enableSession?ticket=${ticket}&adminRoot=${encodeURIComponent(HABANERO_ADMIN_ROOT)}`;
+    res.redirect(loginUrl)
+  }).catch((error) => {
+    logger.error('Error loging into orchestorator: ' + error)
```

Comment:
Use comma instead of converting an error to a string. It might get output as `[object] Object` :)